src/components/BooksList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
```

Comment:
please relocate this component to `src/containers` directory ("container" components should be inside this folder)